### DISCUSSION TITLE
How to model extended-release drug?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1679] [How-Tos]
Hi!
I am trying to build a drug with extended-release formulation which consists of two different release stages. How can I set up this formulation in PK-Sim or MoBi? Is there any tutorial or publications to follow?
Looking forward for your reply!
Best regards,
Greta

**Comment by Gre-ux** [comment]
Hi! I am trying to model this process in MoBi. 
This drug has an initial quick release in the first few hours, and then released slowly in the next few days.
Here is what I am doing: 
<img width="750" alt="1703517333965" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/114388923/f8b21497-1d8b-45fd-af21-a09ea731916b">
I am trying to change Dose in Events Tab to a  function related to time as seen in the picture. Is that right? Cause it's a little difficult for me to figure out the formula and input it in MoBi. Can you give me some advice? Or is there any tutorial or MoBi file that I can follow?
Really looking forward for your help!
Best,

**Comment by tobiasK2001** [comment]
Dear Greta,

just some quick thoughts: maybe it is easier to start in PK-Sim. There you can create and advanced dosing protocol with one Dose event for the quick release and one for the slow release part of your formulation. A bit similar as Hanke et al described it for the Rosuvastatin model ( 10.1007/s11095-021-03109-6). You can combine the fast and slow release process with respective formulations. Formulations and Administration protocols are easier to set up in PK-Sim. Later you can modify the Events BB in Mobi, too. 

Hope this might help you.

Best, Tobias

**Comment by Yuri05** [comment]
You could also approximate your ER formulation in Excel (selecting an appropriate time raster), then create a table formulation in PK-Sim and import your formulation from Excel.

![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/e868a67f-e63c-4e1e-b779-e2c3b3c227a4)

![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/4162a813-aeff-4f4e-ae38-f29a09a64819)
